admin/services/pupil.service.js

```diff
@@ -25,6 +25,19 @@ const pupilService = {
     }
   },
   /**
+   * Fetch one pupil filtered by pupil id and school id
+   * @param pupilId
+   * @param schoolId
+   * @returns {Promise.<*>}
+   */
+  fetchOnePupil: async (pupilId, schoolId) => {
```

Comment:
We should discuss naming at some point.  I prefer to see the service in the call e.g.
`pupilService.fetchOne(id)`

But I would guess from this name that you prefer 
`fetchOnePupil` which I think is less clear as it hides the service.

Reply:
@jon-shipley I prefer your suggestion, thanks. I'll have a refactoring PR next week with this and other improvements.

Reply:
This should really be moved to pupil.data.service module.

Reply:
@jon-shipley  done in the refactoring branch